can-map-define_test.js

```diff
@@ -1565,3 +1571,62 @@ QUnit.test("can.hasKey", function() {
 	vm.attr('lateProp', 'something');
 	equal(canReflect.hasKey(vm, "lateProp"), true, "vm.hasKey('lateProp') true");
 });
+
+QUnit.test("can.getOwnEnumerableKeys", function() {
+	var ParentMap = CanMap.extend({
+		define: {
+			parentNoEnum: {
+				serialize: false,
+				value: 'parent_no'
+			},
+
+			parentEnum: {
+				serialize: true,
+				value: 'parent_yes'
+			},
+
+			parentEnumByDefault: {
+				value: 'maybe'
+			},
+
+			parentEnumGetter: {
+				get: function () {
+					return 'parent_get';
+				}
+			}
+		}
+	});
+
+		var VM = ParentMap.extend({
+		define: {
+			notEnumerable: {
+				serialize: false,
+				value: 'no'
+			},
+
+			enumerableProp: {
+				serialize: true,
+				value: 'yes'
+			},
+
+			enumByDefault: {
+				value: 'maybe'
+			},
+
+			enumGetter: {
+				get: function () {
+					return 'got';
+				}
+			}
+		}
+	});
+
+	var vm = new VM();
+	var getOwnEnumerableKeysSymbol = canSymbol.for("can.getOwnEnumerableKeys");
+	// getOwnEnumerableKeys for inherited defined props
+	deepEqual( vm[getOwnEnumerableKeysSymbol](), [ "enumerableProp", "enumByDefault", "parentEnum", "parentEnumByDefault" ], "vm.getOwnEnumerableKeys()");
```

Comment:
You can use `canReflect.getOwnEnumerableKeys(vm)` so that you don't have to add `can-symbol` as a dependency.